### ISSUE TITLE
Custom ObjectMapper in application affects Kafka UI Dev service

### ISSUE THREAD

**Issue by gitkeerthi** [quarkusio/quarkus#29976]
### Describe the bug

I have custom ObjectMapperCustomizer that I needed to convert all json payloads to use snake_case.

```
@Singleton
public class ObjectMapperCustomizer implements io.quarkus.jackson.ObjectMapperCustomizer {

    public void customize(ObjectMapper mapper) {
        mapper.setSerializationInclusion(NON_NULL);
        mapper.setPropertyNamingStrategy(SNAKE_CASE);
    }

}
```
This apparently affects the Kafka UI running as one of the dev services. How do I isolate kafka dev services from this?

### Expected behavior

The Kafka UI dev service should function as expected

### Actual behavior

Application's custom JsonMappers affects Kafka UI's payload and as a result renders the UI unusable

**Comment by gitkeerthi** [comment]
More details: https://stackoverflow.com/questions/74862723/quarkus-smallrye-kafka-ui-unexpected-error

**Comment by ozangunalp** [comment]
@gitkeerthi Thanks for opening the issue here. The Kafka UI server needs to create its own ObjectMapper instance.

**Comment by gitkeerthi** [comment]
@ozangunalp Is there a workaround you suggest?

**Comment by ozangunalp** [comment]
@gitkeerthi I can't think of any workarounds. But I think the fix can be backported.

**Comment by gsmet** [comment]
The fix will be in 2.15.1.Final, the release is in progress and should hopefully hit Maven Central before the end of the day.

**Comment by gitkeerthi** [comment]
@ozangunalp Sorry to report that the issue still exists in 2.15.1.Final
